### ISSUE TITLE
Found vulnerabilities for xstream and guava

### ISSUE THREAD

**Issue by enozcan** [hazelcast/hazelcast-docker#191]
Daily vulnerability checks fail:
```
---3.x and 4.x---

[0008] DEBUG found 1 vulnerabilities for pkg=Pkg(type=java-archive, name=xstream, version=1.4.13)
[0008] DEBUG   └── vuln='GHSA-mw36-7c6c-q4q2' type='Exact-Direct Match' key='map[language:java]' foundBy='java-matcher'

---4.x only---

[0008] DEBUG found 2 vulnerabilities for pkg=Pkg(type=java-archive, name=guava, version=24.1.1-jre)
[0008] DEBUG   ├── vuln='GHSA-mvr2-9pj6-7w5j' type='Exact-Direct Match' key='map[language:java]' foundBy='java-matcher'
[0008] DEBUG   └── vuln='CVE-2018-10237' type='Fuzzy Match' key='map[cpe:cpe:2.3:*:*:guava:24.1.1-jre:*:*:*:*:java:*:*]' foundBy='java-matcher'
```
3.x: https://github.com/hazelcast/hazelcast-docker/runs/1415841667?check_suite_focus=true
4.x: https://github.com/hazelcast/hazelcast-docker/runs/1415816086?check_suite_focus=true

**Comment by hasancelik** [comment]
Opened issue at [Netflix/eureka](https://github.com/Netflix/eureka) repo about **xstream** vulnerability:
https://github.com/Netflix/eureka/issues/1367

**Comment by hasancelik** [comment]
about guava related one, the vulnerability affects all versions between `11.0` and `24.1.1`, we have multiple transitive dependencies that uses versions from that range:
```
[INFO] +- com.netflix.eureka:eureka-client:jar:1.10.10:compile
[INFO] |  +- com.netflix.netflix-commons:netflix-eventbus:jar:0.3.0:compile
[INFO] |  |  +- com.netflix.netflix-commons:netflix-infix:jar:0.3.0:runtime
...
[INFO] |  |  |  +- (com.google.guava:guava:jar:14.0.1:runtime - omitted for conflict with 16.0)
[INFO] |  |  |  \- (com.google.code.gson:gson:jar:2.1:runtime - omitted for conflict with 2.8.6)
...
[INFO] |  +- com.netflix.servo:servo-core:jar:0.12.21:compile
[INFO] |  |  +- org.slf4j:slf4j-api:jar:1.6.4:compile
[INFO] |  |  \- (com.google.guava:guava:jar:16.0:compile - omitted for conflict with 30.0-jre)
....
[INFO] |  +- com.google.inject:guice:jar:4.1.0:compile
[INFO] |  |  \- (com.google.guava:guava:jar:19.0:compile - omitted for conflict with 16.0)
...
[INFO] +- com.netflix.archaius:archaius-core:jar:0.7.7:compile
[INFO] |  +- (com.google.guava:guava:jar:16.0:compile - scope updated from runtime; omitted for duplicate)
```
This CVE was already [reported at archauis repo](https://github.com/Netflix/archaius/issues/555) two years ago but related fix is not merged yet. To remove `archaius-core` from that list , we can start to use [eureka-client-archaius2](https://github.com/Netflix/eureka/tree/master/eureka-client-archaius2) which utilizes `archaius2`. [With latest release](https://github.com/Netflix/archaius/releases/tag/v2.3.16), they have removed guava dependency from the codebase. 

For other repos like `netflix-commons, netflix-servo`, we need to ping maintainers.

**Comment by hasancelik** [comment]
After some tests and small discussion with @leszko , it seems transitive dependencies are not evaluated by anchore. So we do not need to wait Netflix/eureka PR and worry about other guava dependencies. 

`xstream` vulnerability was fixed via https://github.com/hazelcast/hazelcast-docker/pull/192 
`guava` vulnerability should be fixed at hazelcast OS repo:
https://github.com/hazelcast/hazelcast/pull/17743

**Comment by hasancelik** [comment]
`guava` PR at hazelcast core merged, current `guava` version is `30.0-jre` so closing this issue. If scan builds fail again, you can re-open the issue @enozcan
